pkg/i18n/french.go

```diff
@@ -8,32 +8,32 @@ func frenchSet() TranslationSet {
 		StartingStatus:             "démarrage",
 		StoppingStatus:             "arrêt",
 		PausingStatus:              "mise en pause",
-		RunningCustomCommandStatus: "execution de la commande personalisée",
-		RunningBulkCommandStatus:   "execution de la commande groupée",
+		RunningCustomCommandStatus: "exécution de la commande personalisée",
+		RunningBulkCommandStatus:   "exécution de la commande groupée",
 
 		NoViewMachingNewLineFocusedSwitchStatement: "No view matching newLineFocused switch statement",
 
-		ErrorOccurred:              "Une erreur s'est produite! Veuillez créer un rapport d'erreur sur https://github.com/jesseduffield/lazydocker/issues",
+		ErrorOccurred:              "Une erreur s'est produite ! Veuillez créer un rapport d'erreur sur https://github.com/jesseduffield/lazydocker/issues",
 		ConnectionFailed:           "Erreur lors de la connexion au client docker. Essayez de redémarrer votre client docker",
-		UnattachableContainerError: "Le conteneur ne peut pas être attaché. Vous devez executer le service avec le drapeau 'it' ou bien utiliser `stdin_open: true, tty: true` dans votre fichier docker-compose.yaml",
-		WaitingForContainerInfo:    "Le processus ne peut pas continuer avant que docker donne plus d'informations. Veuillez réessayer dans quelques instants.",
+		UnattachableContainerError: "Le conteneur ne peut pas être attaché. Vous devez exécuter le service avec le drapeau 'it' ou bien utiliser `stdin_open: true, tty: true` dans votre fichier docker-compose.yaml",
```

Comment:
```suggestion
		UnattachableContainerError: "Le conteneur ne peut pas être attaché. Vous devez exécuter le service avec le drapeau '-it' ou bien utiliser `stdin_open: true, tty: true` dans votre fichier docker-compose.yml",
```
Par souci d'homogénéité avec la version anglaise. 😄